README.md

```diff
@@ -70,6 +70,6 @@ var_dump($patternRenderer->getStyleSheets());
 Updating the library
 --------------------
 
-1. Install [Compass](http://compass-style.org/).
+1. Install [Docker](https://www.docker.com/).
 2. Execute `bin/update` to update the `resources` folder from Pattern Lab (you can pass a commit if needed, eg `bin/update baa62f0`).
```

Comment:
This will need to be the full sha, but we'll need to push them first. Need to mention PR tags too.

Reply:
Is the sha stored somewhere? Or is it always passed as an input?